packages/flame_bloc/README.md

```diff
@@ -1,53 +1,109 @@
 # Flame Bloc 🔥🧱
 
-`flame_bloc` adds easy access to blocs/cubits that are available on the widget tree to your Flame
-game and makes it possible for Flame components to listen to state changes to those blocs/cubits.
+`flame_bloc` offers a simple and natural (as in similar to `flutter_bloc`) way to use blocs and
+cubits inside a `FlameGame`.
+
+## ⚠️  IMPORTANT ⚠️  ##
+This README only covers the new (>= 1.4.0) API of flame bloc, for the previous deprecated API check
+[this](./DEPRECATED_README.md).
```

Comment:
I don't think we need this, if they want to read an old version of the readme they can look at the tag for an old version.